pkgs/applications/networking/remote/tlclient/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+
+, alsaLib
+, libX11
+, pcsclite
+}:
+let
+  tlclient-desktop = fetchurl {
+      url = "https://aur.archlinux.org/cgit/aur.git/plain/tlclient.desktop?h=tlclient";
+      sha256 = "1219pqcbzh38ffxlz26pmn2yq6d2r3s0bjn6yaz319gnl9zl39ya";
+  };
```

Comment:
I would prefer that you use `mkDesktopItem` instead and created a proper desktop item for NixOS. Using `copyDesktopItems` can also help adding the desktop item to your derivation without copying the generated desktop file manually. See here for details: https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/X11/opentabletdriver/default.nix

If you're going to use like this, at least pin to a commit so we avoid breakage if the desktop file in Arch changes: https://aur.archlinux.org/cgit/aur.git/plain/tlclient.desktop?h=tlclient&id=fd126edd54a012caa8c9ce47ab8d7fb25e982e74

Reply:
BTW, checking the desktop file, `Exec=/usr/bin/tlclient` wouldn't work in NixOS I think since this is not a valid path to the binary. Also there is a `Version=4.10.0` that needs to be bumped at every upgrade (so it is wrong right now), but either way I think this information about version is irrelevant.

So my recommendation is to really use `mkDesktopItem`.

Reply:
Fixed. BTW I didn't know such tool existed, thanks !